spec/features/visitors/email_confirmation_spec.rb

```diff
@@ -55,7 +55,12 @@
 
   context 'confirmed user is signed in and tries to confirm again' do
     it 'redirects the user to the profile' do
-      sign_up_and_2fa
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
For complex multi-step things like this, I have a hard time remembering which helper methods do which so I do have a tendency to copy-paste (especially since we have many similar and overlapping sign-in helper methods)

However since all 5 lines or so seem to be the exact same in a few files, I agree that this seems like a good candidate to just keep in the support file? I think I agree with @pkarman here.

Re: the return value, I think it's fine if the method returns a user and we throw it out, the tests all have slightly different intentions?